lib/plugins/dot-net.js

```diff
@@ -1,5 +1,8 @@
 import nugetResolver from '../resolver/nuget';
-import { NET_PACKAGE } from '../../packages/helper-grammar-regex-collection/index.js';
+import {
+  NET_PACKAGE,
+  NET_PROJ_PACKAGE,
```

Comment:
This is the format `yarn lint` wanted, even though in other files they're on one line, such as in https://github.com/OctoLinker/browser-extension/blob/d781585917cbca8c86e43b73222ccc6e29114375/lib/plugins/bower-manifest.js#L4